test/testinput/bufr_ncep_satwind_avhrr.yaml

```diff
@@ -0,0 +1,152 @@
+            query: "*/WSPD"
+
+          generatingApplication:
+            query: "*/GQCPRMS/GNAP[1]"
+
+          percentConfidence:
+            query: "*/GQCPRMS/PCCF[1]"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testoutput/bufr_ncep_satwind_avhrr.nc"
+      dimensions:
+        - name: Confidence
+          paths:
+            - "*/GQCPRMS"
+
+      variables:
+        - name: "MetaData/satelliteIdentifier"
+          source: variables/satelliteIdentifier
+          longName: "Satellite Identifier"
+          units: ""
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude (Coarse Accuracy)"
+          units: "degree_north"
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude (Coarse Accuracy)"
+          units: "degree_east"
+
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/satelliteDerivedWindComputationMethod"
+          source: variables/satelliteDerivedWindComputationMethod
+          longName: "Satellite Derived Wind Computation Method"
+          units: ""
+
+        - name: "MetaData/satelliteZenithAngle"
+          source: variables/satelliteZenithAngle
+          longName: "Satellite Zenith Angle"
+          units: "degree"
+
+        - name: "MetaData/originatingCenter"
+          source: variables/originatingCenter
+          longName: "Identification of Originating/Generating Center"
+          units: ""
+
+        - name: "MetaData/satelliteChannelCentreFrequency"
+          source: variables/satelliteChannelCentreFrequency
+          longName: "Satellite Channel Centre Frequency"
+          units: "Hz"
+
+        - name: "QualityMarker/sdmeditSatWindQualMark"
+          source: variables/sdmeditSatWindQualMark
+          longName: "SDMEDIT Satellite Wind Quality Mark"
+          units: ""
+
+        - name: "MetaData/heightAssignmentMethod"
+          source: variables/heightAssignmentMethod
+          longName: "Height Assignment Method"
+          units: ""
+
+        - name: "MetaData/pressure"
+          source: variables/pressure
+          longName: "Pressure"
+          units: "Pa"
+
+        - name: "ObsValue/windDirection"
+          source: variables/windDirection
+          longName: "Wind Direction"
+          units: "deg"
+
+        - name: "ObsValue/windSpeed"
+          source: variables/windSpeed
+          longName: "Wind Speed"
+          units: "m s-1"
+
+        - name: "MetaData/generatingApplication"
+          source: variables/generatingApplication
+          longName: "Generating Application"
+          units: ""
+
+        - name: "MetaData/percentConfidence"
+          source: variables/percentConfidence
+          longName: "Per Cent Confidence"
+          units: "percent"
+
```

Comment:
Instead of sticking them in `MetaData`.

Reply:
I like the idea, though how would I go about implementing this? If I used the following set, does the bottom half (name/source/longName/units) only matter in regards to the order in which they are listed?

```
          windDirection:
            query: "*/WDIR"

          windSpeed:
            query: "*/WSPD"

          windSpeed:
            query: "*/GQCPRMS/GNAP[1]"

          windDirection:
            query: "*/GQCPRMS/GNAP[2]"

          windSpeed:
            query: "*/GQCPRMS/PCCF[1]"

          windDirection:
            query: "*/GQCPRMS/PCCF[2]"
```
So then the bottom would just be:
```
        - name: "ObsValue/windDirection"
          source: variables/windDirection
          longName: "Wind Direction"
          units: "deg"

        - name: "ObsValue/windSpeed"
          source: variables/windSpeed
          longName: "Wind Speed"
          units: "m s-1"

        - name: "GeneratingApplication/windSpeed"
          source: variables/windSpeed
          longName: "Wind Speed Generating Application"
          units: ""

        - name: "GeneratingApplication/windDirection"
          source: variables/windDirection
          longName: "Wind Direction Generating Application"
          units: ""

        - name: "PercentConfidence/windSpeed"
          source: variables/windSpeed
          longName: "Wind Speed Per Cent Confidence"
          units: "percent"

        - name: "PercentConfidence/windDirection"
          source: variables/windDirection
          longName: "Wind Direction Per Cent Confidence"
          units: "percent"
```
My assumption is that this wouldn't work given the repeated sources.

Reply:
Your close, but the variable names (in the exports section) must all have unique names. That is to say they must all be named something different (otherwise we can't tell them apart). So instead `*/GQCPRMS/GNAP[1]` being named `windSpeed` it should be called something like `windSpeedGeneratingApp`. The only one that is named `windSpeed` should be the one thats associated with the `*/WSPD` BUFR query. The same with the other variables.

Reply:
Okay, that was my understanding.

Currently, I have it set as :
          windDirection:
            query: "*/WDIR"

          windSpeed:
            query: "*/WSPD"

          windSpeedGeneratingApplication:
            query: "*/GQCPRMS/GNAP[1]"

          windDirectionGeneratingApplication:
            query: "*/GQCPRMS/GNAP[2]"

          windSpeedPercentConfidence:
            query: "*/GQCPRMS/PCCF[1]"

          windDirectionPercentConfidence:
            query: "*/GQCPRMS/PCCF[2]"

Reply:
Thats perfect.